README.md

```diff
@@ -54,6 +54,9 @@ class DefaultController extends Controller
         $config->setSortCols(['t.id'], ['t.id' => 'asc']);
         $config->setRequest(new Request($request));
         
+        // Throws an UnexpectedValueException when invalid filter column is specified  
```

Comment:
Throws an UnexpectedValueException when invalid filter column, sort column or sort type is specified

Reply:
Good point, edited.